controllers/component_image_controller.go

```diff
@@ -187,119 +145,76 @@ func (r *ComponentReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		return ctrl.Result{}, r.reportError(ctx, component, message)
 	}
 
-	setMetricsTime(componentIdForMetrics, reconcileStartTime)
```

Comment:
Should we remove this metric at all?

Reply:
I didn't remove it, but moved it before creating imagerepository 
https://github.com/konflux-ci/image-controller/pull/126/files#diff-6d10841540c4fbc83a67ba9f310705ea685b3650d4e865fbd8605c052209812aR178